service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordOldPasswordMatchesRuleTest.java

```diff
@@ -93,14 +92,14 @@ public void isValidShouldNotBeValidWhenPasswordChangeDTOIsNull() {
   }
 
   @Test
-  public void isValidShouldNotBeValidWhenUserDetailsIsNull() {
+  public void isValidShouldNotBeValidWhenUserDetailsIsNull() throws UserNotLoggedInException {

```

Comment:
Extend the test cases with the scenario when AuthenticationService throws UserNotLoggedInException.

It should work like the case when user details is null. Modify the production code as well.